exercises/concept/paulas-palindromes/src/Main.elm

```diff
@@ -0,0 +1,52 @@
+module Main exposing (..)
+
+import Browser
+import Html exposing (Html, div, input, text)
+import Html.Attributes exposing (placeholder, value)
+import Html.Events exposing (onInput)
+
+
+
+-- When you have finished this exercise, this file will be an Elm program.
+--
+-- You can copy this file to Ellie (an online Elm sandbox - https://ellie-app.com/)
+-- to see it running and interact with it.
+--
+-- If you are solving the exercise locally (and not through the exercism online editor),
+-- you can additionally run it locally:
+-- - `rm src/Browser.elm` (delete a fake file that we need for the online editor),
+-- - `elm install elm/browser` (install the real version of the fake)
+-- - `npx elm-watch@beta hot` (compile the code, hot reload and serve)
+-- - Then go to the url in the output to view the application (localhost://53529 or similar)
```

Comment:
I don't think we need Elm Watch here, it's simpler to do
```suggestion
-- - `rm src/Browser.elm` (delete a fake file that we need for the online editor),
-- - `elm install elm/browser` (install the real version of the fake)
-- - `elm reactor` (compile the code, hot reload and serve)
-- - go to `http://localhost:8000` in a browser to view the application
```

Reply:
I tried elm reactor and elm-watch. I think elm-watch is better / simpler.

Reply:
probably, but it's an extra dependency, if it's not needed, we shouldn't push it on students.

Reply:
Elm reactor is confusing, as it initially shows the directory structure, and you have to browse to the file you want. elm-watch is also a lot more likely to be what students will want as they write more Elm, so good to start them on the right path. And students will probably want to tinker with the app once they have it running (and we should encourage that), so the hot reload is useful too.